content/v2.0/reference/cli/influx/config/list.md

```diff
@@ -0,0 +1,28 @@
+---
+title: influx config list
+description: The 'influx config list' command lists all InfluxDB connection configurations.
+menu:
+  v2_0_ref:
+    name: influx config list
+    parent: influx config
+weight: 201
+---
+
+The `influx config list` command lists all InfluxDB connection configurations in the `config` file (by default, stored at `~/.influxdbv2/config`). Each connection configuration includes a url, authentication token, and active setting. By default, the list includes an asterisk next to the active connection configuration. InfluxDB reads the token from the active connection configuration, so you don't have to manually enter a token to log into InfluxDB.
```

Comment:
- ~By default, the list includes an asterisk next to the active connection configuration.~ -> An asterisk (`*`) indicates the active configuration.
- "InfluxDB reads the token from the active connection configuration, so you don't have to manually enter a token to log into InfluxDB." - This feels out of place here. I think this should be on the `influx config` landing page.

Reply:
moved last statement to landing page